.github/workflows/jdk8-release.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and push the Docker images for JDK 8
+
+on:
+  push:
+    tags:
+      - "jdk8-[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Use the dedicated tag to trigger the release action, as mentioned in the PR description.